server/route.go

```diff
@@ -2333,8 +2479,10 @@ func (s *Server) updateRouteSubscriptionMap(acc *Account, sub *subscription, del
 		return
 	}
 
-	// Create the fast key which will use the subject or 'subject<spc>queue' for queue subscribers.
-	key := keyFromSub(sub)
+	// Create the fast key which will use the subject or '[origin]<spc>subject<spc>queue'
```

Comment:
This comment is wrong. I did change the way this key is constructed (see keyFromSubWithOrigin). Will update tomorrow.